modules/monitoring-about-creating-alerting-rules-for-user-defined-projects.adoc

```diff
@@ -22,3 +24,6 @@ This label forces only the Prometheus instance deployed in the `openshift-user-w
 If an alerting rule has this label, your alerting rule can use only those metrics exposed by your user-defined project.
 Alerting rules you create based on default platform metrics might not trigger alerts.
 ====
+
+* By default, when you create an alerting rule, a project label is enforced on it even if a rule with the same name exists in another project. You can create alerting rules that are not bound to their project of origin by configuring a project in the `user-workload-monitoring-config` config map. For more information, see "Creating cross-project alerting rules for user-defined projects".
```

Comment:
technically the system enforces the value of the `namespace` label, e.g. "project" isn't a recognized label for metrics. Also it might be more appropriate to move this part next to lines 13-18?